modules/web-mercator/src/web-mercator-utils.js

```diff
@@ -204,10 +204,11 @@ export function getProjectionParameters({
   const pitchRadians = pitch * DEGREES_TO_RADIANS;
   const halfFov = Math.atan(0.5 / altitude);
   const topHalfSurfaceDistance =
-    (Math.sin(halfFov) * altitude) / Math.sin(Math.PI / 2 - pitchRadians - halfFov);
+    (Math.sin(halfFov) * altitude) /
+    Math.sin(Math.min(Math.max(Math.PI / 2 - pitchRadians - halfFov, 0.01), Math.PI - 0.01));
```

Comment:
Matches https://github.com/mapbox/mapbox-gl-js/blob/f957713a2806df1f417d84b6e99d3386cc07f046/src/geo/transform.js#L691

Reply:
An ever changing target...

Reply:
This is not really an alignment issue, i.e. if Mapbox ever changes their implementation we do not have to change it. `0.01` technically can be any arbitrary small number above zero.